### PR TITLE
[DEVX-1068] Add docs for browserArgs in puppeteer runner

### PR DESCRIPTION
**Pull request by tianfeng92**
**Thank you for contributing to the `sauce-docs` project!**
**A well described pull request helps maintainers quickly review and merge your change**

Before submitting your PR, please check our [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD)
guidelines, applied for this repository. Avoid large PRs, help reviewers by making them as simple 
and short as possible.

```
browserArgs:
  - "--disable-features=site-per-process"
  - "--start-fullscreen"
```

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD) document.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by valentyn88 — APPROVED** [review]

**Review by alexplischke — APPROVED** [review]

File: docs/testrunner-toolkit/configuration/puppeteer.md [hunk 1]
```diff
@@ -385,6 +385,17 @@ One or more paths to the puppeteer test files to run for this suite. Regex value
     testMatch: ["**/*.js"]
 ```
 
+---
+
+### `browserArgs`
+<p><small>| OPTIONAL | ARRAY |</small></p>
+
+Pass browser args as the launch options of puppeteer.
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/configuration/puppeteer.md:393]:
```suggestion
Pass flags to configure how Puppeteer launches the selected browser. Review supported flags for [Chrome/Chromium](https://peter.sh/experiments/chromium-command-line-switches/) or [Firefox](The link from Puppeteer is broken).
```
@tianfeng92  -- am I understanding this property correctly? I looked at this page: https://pptr.dev/#?product=Puppeteer&version=v10.1.0&show=api-puppeteerlaunchoptions

Comment (sweeneyskirt-sl) [reply docs/testrunner-toolkit/configuration/puppeteer.md:393]: I can't really find anything for Firefox that provides a list of valid arguments. The closest page I found was here: https://developer.mozilla.org/en-US/docs/Web/WebDriver/Capabilities/firefoxOptions, which doesn't really provide a "list" of potential values. Maybe we just provide the Chrome link?

Comment (tianfeng92) [reply docs/testrunner-toolkit/configuration/puppeteer.md:393]: I think it would be enough to only provide the chrome link currently 🤔

Comment (tianfeng92) [reply docs/testrunner-toolkit/configuration/puppeteer.md:393]: Let me search that a bit. I don't think https://pptr.dev/#?product=Puppeteer&version=v10.1.0&show=api-puppeteerlaunchoptions is providing `browser args`. It's just launching options of puppeteer.

Comment (tianfeng92) [reply docs/testrunner-toolkit/configuration/puppeteer.md:393]: found one for firefox http://www.devdoc.net/web/developer.mozilla.org/en-US/docs/Mozilla/Command_Line_Options.html
But it's from third-party web archive provider. Is it fine to put that in our docs?

Comment (sweeneyskirt-sl) [reply docs/testrunner-toolkit/configuration/puppeteer.md]: Oh...sorry. I was reviewing the wrong commit. I would ask @KunalJain if it is fine to include that link.

**Review by sweeneyskirt-sl — APPROVED** [review]

**Pull request merged**